auth-web/tests/unit/components/AccountBusinessType.spec.ts

```diff
@@ -63,10 +64,55 @@ describe('AccountBusinessType.vue', () => {
       mocks: { $t
       }
     })
-    await wrapper.setData({ isLoading: false, isBusinessAccount: true })
-    wrapper.find("[data-test='radio-business-account-type']").trigger('click')
+    await wrapper.setData({ isLoading: false, orgType: AccountType.BUSINESS })
+    await wrapper.vm.handleAccountTypeChange(AccountType.BUSINESS)
     await flushPromises()
     expect(wrapper.find("[data-test='input-branch-name']").isVisible()).toBeTruthy()
     expect(wrapper.find("[data-test='business-account-type-details']").exists()).toBeTruthy()
   })
+
+  it('renders the government agency radio button when not editing and not a current government org', async () => {
+    const $t = () => ''
+    wrapper = mount(AccountBusinessType, {
+      localVue,
+      vuetify,
+      propsData: {
+        govmAccount: false,
```

Comment:
unit test for govmAccount: true?

Reply:
added
![image](https://github.com/bcgov/sbc-auth/assets/16507223/15ea595c-b1a7-45a8-b081-e241c7ba5097)